### PR TITLE
Change links to have full path names in TOC.md for distribute documents

### PR DESCRIPTION
**Pull request by journey2w**
- Fix link errors.

### CODE REVIEW

File: docs/application/distribute/TOC.md [hunk 1]
```diff
@@ -1,31 +1,29 @@
...
 # Validation
 ## Guidelines
 ### [Introduction] (validation/guidelines/introduction.md)
-### [Application Validation Process](validation/guidelines/application-validationn-process.md)
-### [Tizen Application Compliance](validation/guidelines/tizen-application-compliance.md)
-### [Age Rating Guide](validation/guidelines/age-rating-guide.md)
-### [Usability Evaluation Guide](validation/guidelines/usability-evaluation-guide.md)
-## [Tizen Validation Policy](validation/tizen-validationn-policy.md)
-## [Self-Check List](validation/self-check-list.md)
+### [Application Validation Process](/application/distribute/validation/guidelines/application-validationn-process.md)
```
Comment (mijin85cho) [inline docs/application/distribute/TOC.md]: application-validationn-process.md -> application-validation-process.md

File: docs/application/distribute/TOC.md [hunk 2]
```diff
@@ -1,31 +1,29 @@
...
+### [Tizen Application Compliance](/application/distribute/validation/guidelines/tizen-application-compliance.md)
+### [Age Rating Guide](/application/distribute/validation/guidelines/age-rating-guide.md)
+### [Usability Evaluation Guide](/application/distribute/validation/guidelines/usability-evaluation-guide.md)
+## [Tizen Validation Policy](/application/distribute/validation/tizen-validationn-policy.md)
```
Comment (mijin85cho) [inline docs/application/distribute/TOC.md]: tizen-validationn-policy.md -> tizen-validation-policy.md

File: docs/application/distribute/how-to-distribute/overview.md [hunk 3]
```diff
@@ -2,12 +2,12 @@
...
-| [**Sales**](../how-distribute/sales.md) | - [Country & Price](../how-distribute/sales.md#Country) <br/>- [Availability Date](../how-distribute/sales.md#Availability) <br/>- [Category](../how-distribute/sales.md#Category) <br/>- [Age Restriction](../how-distribute/sales.md#Restriction) |
-| [**Display**](../how-distribute/display.md) |- [Application Detail](../how-distribute/display.md#ApplicationDetails) <br/>- [Icon Image](../how-distribute/display.md#IconImage) <br/>- [Screenshots](../how-distribute/display.md#Screenshots) <br/>- [YouTube URL](../how-distribute/display.md#YouTubeURL) <br/>- [Banner](../how-distribute/display.md#Banner) <br/>- [Support](../how-distribute/display.md#Support) |
-| [**Item**](../how-distribute/item.md) | - [In App Purchase Type & Item Group ID](../how-distribute/item.md) |
+| [**Binary**](../how-to-distribute/binary.md) | - [Application Title (Default Language)](../how-to-distribute/binary.md#application-t) <br>- [Binary File and Supported Device](../how-to-distribute/binary.md#binary-file) <br>- [Expansion File Upload](../how-to-distribute/binary.md#expansion) <br>- [Validation Information](../how-to-distribute/binary.md#validation) |
```
Comment (mijin85cho) [inline docs/application/distribute/how-to-distribute/overview.md]: `application-t` -> `application-title`

File: docs/application/distribute/how-to-distribute/overview.md [hunk 4]
```diff
@@ -2,12 +2,12 @@
...
+| [**Sales**](../how-to-distribute/sales.md) | - [Country & Price](../how-to-distribute/sales.md#country-price) <br>- [Availability Date](../how-to-distribute/sales.md#availability-date) <br>- [Category](../how-to-distribute/sales.md#category) <br>- [Age Restriction](../how-to-distribute/sales.md#age-restriction) |
```
Comment (mijin85cho) [inline docs/application/distribute/how-to-distribute/overview.md]: It looks like sales.md files doesn't have `<a name="country-price"></a>`,  `<a name="availability-date"></a>` or `<a name="category"></a>`

File: docs/application/distribute/how-to-distribute/overview.md [hunk 5]
```diff
@@ -2,12 +2,12 @@
...
+| [**Display**](../how-to-distribute/display.md) |- [Application Detail](../how-to-distribute/display.md#application-details) <br>- [Icon Image](../how-to-distribute/display.md#icon-image) <br>- [Screenshots](../how-to-distribute/display.md#screenshots) <br>- [YouTube URL](../how-to-distribute/display.md#youtube-URL) <br>- [Banner](../how-to-distribute/display.md#banner) <br>- [Support](../how-to-distribute/display.md#support) |
```
Comment (mijin85cho) [inline docs/application/distribute/how-to-distribute/overview.md]: It looks like `#youtube-URL` is not working.

File: docs/application/distribute/how-to-distribute/sales.md [hunk 6]
```diff
@@ -1,38 +1,40 @@
...
+  5. In countries where the Tier-method of Phone Bills is supported, when an auto-converted price is less than the standard price of USD 5, the price charged will be the closest Tier price, based on the auto-converted figure (+/- 5% of the auto-converted amount is allowed).
+  6. When you enter a price for applications, an open-lock icon will be shown. When you click the lock-shaped icon to change it to a locked icon, the set price cannot be changed. Also, you cannot change the price for the country when you click the [Auto convert local currency] button. If you need to change the price, simply click on the lock-shaped icon.
+  
+     [![img](media/sales2-330x99.png)](media/sales2.png)
```
Comment (mijin85cho) [inline docs/application/distribute/how-to-distribute/sales.md]: This image tag is strange.

File: docs/application/distribute/how-to-distribute/sales.md [hunk 7]
```diff
@@ -1,38 +1,40 @@
...
-- Set the sales period for applications in Availability Date. You can set that sales start immediately after validation and approval have been granted by selecting the checkbox. If you need to set a specific period, you can click the calendar icon to select dates.
 ![img](media/distribution_sales02.jpg)
 
+Set the sales period for applications in Availability Date. You can set that sales start immediately after validation and approval have been granted by selecting the checkbox. If you need to set a specific period, you can click the calendar icon to select dates.
+
 
 ## Category
 
-- Set the sales period for applications in Availability Date. You can set that sales start immediately after validation and approval have been granted by selecting the checkbox. If you need to set a specific period, you can click the calendar icon to select dates.
 [![img](media/sales4-630x90.png)](media/sales4.png)
```
Comment (mijin85cho) [inline docs/application/distribute/how-to-distribute/sales.md]: This image tag is strange.

File: docs/application/distribute/how-to-distribute/sales.md [hunk 8]
```diff
@@ -1,38 +1,40 @@
...
 ## Age Restriction
 
-- In Age Restriction, select All, 12+, 16+ or 18+.
-- When you select Game as the Category and Korea Republic of or Brazil as the Country for Sales, rating information must be entered. A certificate of age restriction must be issued by an external certification authority. Age restrictions will be approved when the appropriate document is provided. You must therefore upload the appropriate document on age restriction issued from the relevant country. When you upload a GRB certificate for Korea, or MJ/DEJUS certificate for Brazil, select the age restriction that matches the document. If there is no certificate or a certificate cannot be issued, deselect the country from the list of countries where the application will be sold.
- 
-   [![img](media/sales5-730x137.png)](media/sales5.png)
- 
- 
\ No newline at end of file
+![img](media/sales5-730x137.png)](media/sales5.png)
```
Comment (mijin85cho) [inline docs/application/distribute/how-to-distribute/sales.md]: This image tag is strange.

File: docs/application/distribute/how-to-distribute/overview.md (hunk 3 above)
Comment (journey2w) [reply docs/application/distribute/how-to-distribute/overview.md]: done

File: docs/application/distribute/how-to-distribute/overview.md (hunk 4 above)
Comment (journey2w) [reply docs/application/distribute/how-to-distribute/overview.md]: "## Availability Date" is the 2nd title. When markdown file is built, boost convert title to HTML tag with "id" property.
So...we dont need to add <a name="xxx"><a> when title and id are same.

File: docs/application/distribute/how-to-distribute/overview.md (hunk 5 above)
Comment (journey2w) [reply docs/application/distribute/how-to-distribute/overview.md]: Done

File: docs/application/distribute/how-to-distribute/sales.md (hunk 7 above)
Comment (journey2w) [reply docs/application/distribute/how-to-distribute/sales.md]: I think there is no problem. The image has a link to show a bigger image. The compressed image is too small so user can't read the contents. we can provide a bigger image by clicking a image. TD site provides this kinds of feture by Drupal.

File: docs/application/distribute/how-to-distribute/sales.md (hunk 8 above)
Comment (journey2w) [reply docs/application/distribute/how-to-distribute/sales.md]: Same as above.

File: docs/application/distribute/TOC.md (hunk 2 above)
Comment (journey2w) [reply docs/application/distribute/TOC.md]: done

File: docs/application/distribute/TOC.md (hunk 1 above)
Comment (journey2w) [reply docs/application/distribute/TOC.md]: done

File: docs/application/distribute/how-to-distribute/sales.md (hunk 8 above)
Comment (mijin85cho) [reply docs/application/distribute/how-to-distribute/sales.md]: The image is shown in this way on github.

image

File: docs/application/distribute/how-to-distribute/sales.md (hunk 6 above)
Comment (journey2w) [reply docs/application/distribute/how-to-distribute/sales.md]: I think there is no problem. The image has a link to show a bigger image. The compressed image is too small so user can't read the contents. we can provide a bigger image by clicking a image. TD site provides this kinds of feture by Drupal.

File: docs/application/distribute/how-to-distribute/sales.md (hunk 8 above)
Comment (journey2w) [reply docs/application/distribute/how-to-distribute/sales.md]: The left [ was missed. fixed.

**Comment by journey2w** [comment]
All reviewed errors were fixed.

**Pull request merged**